.github/workflows/test.yml

```diff
@@ -35,49 +33,34 @@ jobs:
 
     name: "Test: PHP ${{ matrix.php }}"
     continue-on-error: ${{ matrix.php == '8.2' }}
+    runs-on: ubuntu-latest
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Set coverage variable
-        id: set_cov
-        run: |
-          if [ ${{ matrix.coverage }} == "true" ]; then
-            echo '::set-output name=COV::xdebug'
-          else
-            echo '::set-output name=COV::none'
-          fi
-
-      - name: Install PHP
+      - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
           ini-values: error_reporting=-1, display_errors=On, log_errors_max_len=0
-          coverage: ${{ steps.set_cov.outputs.COV }}
+          coverage: ${{ matrix.coverage && 'xdebug' || 'none' }}
```

Comment:
Voila! 💜

Reply:
While I agree this works, I prefer readability and making it intuitive what's happening above brevity to make these kind of scripts easier to read for people less familiar with the syntax.

Reply:
> for people less familiar with the syntax

I do not work for those people :(

Reply:
> I do not work for those people :(

In open source you generally don't work "for" people in the first place.

However, keeping an open source project open to contributions from people with less experience than yourself is IMO always a good idea.